news/4.33/platform_isv.html

```diff
@@ -38,6 +38,24 @@ <h2>Platform and Equinox API</h2>
   <tr>
     <td id="Platform" class="section" colspan="2"><h2>Platform Changes</h2></td>
   </tr>
+  <tr id="IFileAPI"> <!-- https://github.com/eclipse-platform/eclipse.platform/pull/1397 -->
+    <td class="title">IFile API for create/read/write</td>
+    <td class="content">
+		Since 3.21 <code>org.eclipse.core.resources</code> defines new methods for:
```

Comment:
```suggestion
		<p>The <code>org.eclipse.core.resources</code> plug-in defines the following new methods:</p>
```
Given these are 4.33 notes, the version number is just a bit confusing.